MKMProvider/requests.js

```diff
@@ -23,7 +23,10 @@ function getCardFromMkm(cardName) {
         });
     })
     .then(res => {
-        return res.map((card, i, res) => {
+        return res.filter(function(card) {
```

Comment:
Can you use here arrow function ? 
E.g. `res.filter(card => condition1 && condition2)`